google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -250,6 +258,21 @@ class ConnectionWorker implements AutoCloseable {
   private static String projectMatching = "projects/[^/]+/";
   private static Pattern streamPatternProject = Pattern.compile(projectMatching);
 
+  private Meter writeMeter;
+  static AttributeKey<String> telemetryKeyWriterId = AttributeKey.stringKey("writer_id");
```

Comment:
What's writer id?  Is this a UUID for the StreamWriter or JsonStreamWriter?  Does JsonStreamWriter have its own ID, or the ID of the writer that it extends/wraps?

Reply:
I have switched to using stream name.